client/app/components/message/MessageForm.jsx

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-var React = require('react');
+var React = require('tux/React');
 
-var MessageForm = React.createClass({
+var MessageForm = React.createOwneeClass({
   propTypes: {
     createMessage: React.PropTypes.func,
```

Comment:
With the ownerProps will the createMessage and updateMessage functions be on a props of a Message component?